### PR TITLE
In 132797653 remove her

### PR DESCRIPTION
**Pull request by NateOSI**

### CODE REVIEW

File: app/controllers/screenings_controller.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ class ScreeningsController < ApplicationController # :nodoc:
   def create
     new_screening = Screening.new(reference: LUID.generate.first)
     @screening = ScreeningRepository.create(new_screening)
-    redirect_to edit_screening_path(@screening)
+    redirect_to edit_screening_path(@screening.id)
```
Comment (pemaOSI) [inline app/controllers/screenings_controller.rb]: do we have to explicitly specify id to path?

File: spec/features/show_screening_spec.rb [hunk 2]
```diff
@@ -23,8 +23,7 @@
         street_address: '123 fake st',
         city: 'Springfield',
         state: 'NY',
-        zip: 12_345,
-        person_id: nil
```
Comment (pemaOSI) [inline spec/features/show_screening_spec.rb]: is it because we have join table?

**Comment by pemaOSI** [comment]
Looks good. Left some comments.

File: app/views/screenings/edit.html.haml [hunk 3]
```diff
@@ -1,7 +1,6 @@
 %h1
   = I18n.t(:edit, scope: [:views, :screening], reference: @screening.reference)
-
-= form_for @screening do |f|
+= form_for @screening, url: screening_path(@screening.id), method: :put do |f|
```
Comment (NateOSI) [inline app/views/screenings/edit.html.haml]: nah it doesn't

File: spec/features/show_screening_spec.rb (hunk 2 above)
Comment (NateOSI) [inline spec/features/show_screening_spec.rb]: yeah

**Pull request merged**